setup.cfg

```diff
@@ -169,12 +169,16 @@ doctest_subpackage_requires =
 
 [flake8]
 max-line-length = 88
+select = E,F,W
 exclude = extern,*parsetab.py,*lextab.py
-extend-ignore = E203
+extend-ignore = E1,E2,E501,E711,E721,E731,E741,F403,F821,F841,W5
```

Comment:
`E203` seems to be missing here. Is this covered by the `E2` or will it need to be added back?

Reply:
Yeah. It's covered by E2.

Reply:
Alright, `E203` is required for `flake8` to work with `black`. Should we explicitly require it so that if we revisit this list at a later date so this does not get forgotten about?

Reply:
I think it's subsumed in the larger E2 category, which are all obviated by ``black``.  Now that we have black, we *must* ignore E203 and *can* ignore all the rest. So I've bulk ignored the whole category.

Reply:
The main reason I could think to separate it out is for flake tools in development environments. But since we have pre-commit (locally and on CI), `black` will fix the E2 violations.

Reply:
It is probably fine to include things that black will fix, indeed especially for people like @pllim and me using flake while editing.